src/main/scala/nodes/nlp/POSTagger.scala

```diff
@@ -0,0 +1,16 @@
+package nodes.nlp
+
+import epic.sequences.{CRF, Segmentation, TaggedSequence}
+import epic.trees.AnnotatedLabel
+import workflow.Transformer
+import breeze.linalg.Counter2
+import org.apache.spark.rdd.RDD
+
+case class POSTagger(@transient tagger: CRF[AnnotatedLabel, String])
```

Comment:
Could you add some doc strings that explain the format of the input and output? E.g. It looks like now it assumes that a "sentence" is input as an `Array[String]` and the return is a "Tagged Sentence" where the part of speech comes first and the original string comes second. Should part-of-speech be an `Enum`? At least in English, we have a limited and well defined set of them.

Also, does it make more sense for the result to have a return type other than `(String, String)`? If I think about this in the context of other NLP methods I wonder if there's a richer data structure that makes sense in that domain.